_components/utilities/layout-grid.md

```diff
@@ -4,350 +4,504 @@ layout: styleguide
+<div markdown="1">
+```html
+{{ grid-gutters | strip }}
+```
+</div>
+  </div>
+</div>
+
+<h3>Large gutter</h3>
+<p class="font-sans-6 measure-3"><code class="docs-inline-code">.grid-gap-lg</code> adds a grid-gap between grid-columns in the grid-row, to a value set as <code class="docs-inline-code">$theme-column-gap-large</code> in settings (32px default).</p>
+{{ grid-markers }}
+
+<div class="docs-grid-example">
+{% capture grid-gutters-lg %}
+<div class="grid-row grid-gap-lg">
+  <div class="grid-col-4">
+    <div>.grid-col-4</div>
+  </div>
+  <div class="grid-col-4">
+    <div>.grid-col-4</div>
+  </div>
+  <div class="grid-col-4">
+    <div>.grid-col-4</div>
+  </div>
+</div>
+{% endcapture %}
+{{ grid-gutters-lg }}
+</div>
+
+<div class="usa-accordion-bordered usa-code-sample margin-top-4">
+  <button class="usa-accordion-button" aria-controls="code-gutters-lg" aria-expanded="true">Code</button>
+  <div id="code-gutters-lg" class="usa-accordion-content">
+<div markdown="1">
+```html
+{{ grid-gutters-lg | strip }}
+```
+</div>
+  </div>
+</div>
+
+<div markdown="1" class="font-sans-6 measure-3">
+<h2 id="sass-mixins">Sass mixins</h2>
+When using the source Sass files, you have the option of using Sass variables and mixins to create custom, semantic, and responsive page layouts. Our predefined grid classes use these same variables and mixins to provide a whole suite of ready-to-use classes for fast responsive layouts.
+
+### Variables
+Variables and maps determine the number of columns, the gutter width, and the media query point at which to begin floating columns. We use these to generate the predefined grid classes documented above, as well as for the custom mixins listed below.
+
+```scss
+$theme-column-gap:     16px;
+$theme-column-gap-lg:  32px;
+
+// Change the default sizes of breakpoints
+$uswds-spacing:(
+  large: (
+    'card':            grid-units(20),  // 160px
+    'card-lg':         grid-units(30),  // 240px
+    'mobile':          grid-units(40),  // 320px
+  ),
+  larger: (
+    'mobile-lg':       grid-units(60),  // 480px
+    'tablet':          grid-units(80),  // 640px
+    'tablet-lg':       grid-units(110), // 880px
+  ),
+  largest: (
+    'desktop':         grid-units(128), // 1024px
+    'desktop-lg':      grid-units(150), // 1200px
+    'widescreen':      grid-units(175), // 1400px
+  ),
+);
+
+// Turn on or off breakpoints
+$theme-output-breakpoints: (
+  'card':              false,   // 160px
+  'card-lg':           false,   // 240px
+  'mobile':            false,   // 320px
+  'mobile-lg':         true,    // 480px
+  'tablet':            true,    // 640px
+  'tablet-lg':         false,   // 800px
+  'desktop':           true,    // 1040px
+  'desktop-lg':        false,   // 1200px
+  'widescreen':        false,   // 1400px
+);
+
+$container-max-widths: $desktop;
+```
+
+### Mixins
+Mixins are used in conjunction with the grid variables to generate semantic CSS for individual grid columns.
```

Comment:
- are used --> can be used
- generate semantic CSS for individual grid columns --> add grid functionality to semantic component Sass

Reply:
Updated in befa812.